src/reducers/versions.tsx

```diff
@@ -1523,17 +1542,33 @@ export const createReducer = ({
       }
       case getType(actions.setCurrentVersionId): {
         const { versionId } = action.payload;
-        let { expandedPaths } = state;
+        let { expandedPaths, versionInfo } = state;
+        const { selectedPath, visibleSelectedPath } = state;
         const version = getVersionInfo(state, versionId);
 
         if (version && expandedPaths) {
           expandedPaths = _getPrunedExpandedPaths(expandedPaths, version);
         }
 
+        // TODO: remove this conditional statement once no code depends
+        // on `selectedPath` and `visibleSelectedPath` in version
+        // See https://github.com/mozilla/addons-code-manager/issues/1258
+        if (version) {
+          versionInfo = {
+            ...versionInfo,
+            [version.id]: {
+              ...version,
+              selectedPath: selectedPath || version.selectedPath,
+              visibleSelectedPath,
```

Comment:
It seems that I do not need `visibleSelectionPath || version.visibleSelectedPath`, so I changed it to `visibleSelectedPath`.